staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition/strategy.go

```diff
@@ -48,6 +50,21 @@ func (strategy) NamespaceScoped() bool {
 	return false
 }
 
+// GetResetFields returns the set of fields that get reset by the strategy
+// and should not be modified by the user.
+func (strategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
+	fields := map[fieldpath.APIVersion]*fieldpath.Set{
+		"apiextensions.k8s.io/v1": fieldpath.NewSet(
```

Comment:
this should work, i don't know why crds don't work yet.
the test fails like this:
```
    --- FAIL: TestApplyResetFields/apiextensions.k8s.io/v1,_Resource=customresourcedefinitions (0.02s)
        resetFields_test.go:138: Failed to apply: Apply failed with 1 conflict: conflict with "apply.test" using apiextensions.k8s.io/v1: .status.conditions
    --- FAIL: TestApplyResetFields/apiextensions.k8s.io/v1beta1,_Resource=customresourcedefinitions (0.03s)
        resetFields_test.go:138: Failed to apply: Apply failed with 1 conflict: conflict with "apply.test" using apiextensions.k8s.io/v1: .status.conditions
```
Note the conflict, it's with `apply.test` not the actual manager in the test. So something else is going on.

Reply:
Figured it out, test skips crds for now :-/
https://github.com/kubernetes/kubernetes/pull/92809#discussion_r479782666

The crd status gets changed by the system before the test applies its own status. So we get a conflict.